src/View/App.elm

```diff
@@ -98,26 +104,36 @@ homepageView model =
                 ]
 
 
-view : Model -> Html Msg
+type alias Document msg =
+    { title : String
+    , body : List (Html msg)
+    }
+
+
+view : Model -> Document Msg
 view model =
-    div [ class "container-fluid" ]
-        [ errorsListView model
-        , case (List.head model.clients) of
-            Just client ->
-                homepageView model
-
-            Nothing ->
-                authView model
-        , case model.viewer of
-            Just viewer ->
-                viewerView viewer
-
-            Nothing ->
-                text ""
-        , case model.confirm of
-            Nothing ->
-                text ""
-
-            Just confirm ->
-                Common.confirmView confirm
+    { title = "Tooty"
+    , body =
+        [ div [ class "container-fluid" ]
+            [ errorsListView model
+            , case List.head model.clients of
+                Just client ->
+                    homepageView model
+
+                Nothing ->
+                    authView model
+            , case model.viewer of
+                Just viewer ->
+                    viewerView viewer
+
+                Nothing ->
+                    text ""
+            , case model.confirm of
+                Nothing ->
```

Comment:
Maybe.withDefault ?

Reply:
Nope, we don't want to display the confirm view if `confirm` is Nothing